auctions/dutch-auction/project/tests/utils.rs

```diff
@@ -0,0 +1,177 @@
+use fuels::{prelude::*, types::Identity};
+
+// Load abi from json
+abigen!(Contract(
+    name = "DutchAuction",
+    abi = "out/debug/dutch-auction-abi.json"
+));
+
+pub async fn get_contract_instance() -> (DutchAuction, WalletUnlocked) {
+    // Launch a local network and deploy the contract
+    let config = Config {
+        manual_blocks_enabled: true, // Necessary so the `produce_blocks` API can be used locally
+        ..Config::local_node()
+    };
+    let mut wallets = launch_custom_provider_and_get_wallets(
+        WalletsConfig::new_multiple_assets(
+            2,
+            vec![
+                AssetConfig {
+                    id: AssetId::new([0; 32]),
+                    num_coins: 1,
+                    coin_amount: 10000,
+                },
+                AssetConfig {
+                    id: AssetId::new([1; 32]),
+                    num_coins: 1,
+                    coin_amount: 10000,
+                },
+            ],
+        ),
+        Some(config),
+        None,
+    )
+    .await;
+    let wallet = wallets.pop().unwrap();
+
+    let id = Contract::deploy(
+        "./out/debug/dutch-auction.bin",
+        &wallet,
+        DeployConfiguration::default(),
+    )
+    .await
+    .unwrap();
+
+    (DutchAuction::new(id, wallet.clone()), wallet)
+}
+
+pub async fn active_auctions_of_author(instance: &DutchAuction, author: Identity) -> Vec<u64> {
```

Comment:
I'd create a `setup.rs` for the setup and an interface dir which splits the content into core / info functionality. Take a look at the other apps for examples of how to do this